config/examples/vmservicescrape-metrics-server.yml

```diff
@@ -0,0 +1,49 @@
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMServiceScrape
+metadata:
+  name: metrics-server
+  namespace: kube-system
+spec:
+  namespaceSelector:
+    matchName:
+    - kube-system
+  selector:
+    matchLabels:
+      kubernetes.io/name: Metrics-server
+  endpoints:
+  - port: https
```

Comment:
I've tried latest metrics-server release, and there is no endpoint port name at their configuration. So i had to replace port with `targetPort: 443`. 

 Did you faced the same problem? 0.3.7 release of metrics-server.

 `kubectl -n kube-system get endpoints metrics-server -o yaml` returned:
```bash
apiVersion: v1
kind: Endpoints
metadata:
  creationTimestamp: "2020-09-12T23:31:26Z"
  labels:
    addonmanager.kubernetes.io/mode: Reconcile
    kubernetes.io/cluster-service: "true"
    kubernetes.io/name: Metrics-server
  name: metrics-server
  namespace: kube-system
  resourceVersion: "4011249"
  selfLink: /api/v1/namespaces/kube-system/endpoints/metrics-server
  uid: 3794d5c6-6d55-4118-8b6b-fa5ba4ea6066
subsets:
- addresses:
  - ip: 10.0.0.5
    nodeName: gke-cluster-1-pool-1-preem-d6699a17-jxmr
    targetRef:
      kind: Pod
      name: metrics-server-v0.3.6-547dc87f5f-4568r
      namespace: kube-system
      resourceVersion: "4000294"
      uid: acb418b1-5d4d-4619-9108-b4817aab6105
  ports:
  - port: 443
    protocol: TCP
```

 name for port is missing, probably they changed behaviour or i missed something.

Reply:
yeah this depends on distribution usually when you setup k8s with some tool like k3s, kubeadm is will deploy it for you. I will change it to 433 to have it explicit

Reply:
Also I have created https://github.com/kubernetes-sigs/metrics-server/pull/585 so let's see we can revert back to https when this is fixed.